### PR TITLE
Add Puppet scripts for wso2-obbi worker and dashboard profiles

### PR DESCRIPTION
**Pull request by Dinushi**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: modules/ob_common/manifests/params.pp [hunk 1]
```diff
@@ -30,21 +30,41 @@
   $jdk_name     = 'jdk1.8.0_161'
   $java_home    = "${java_dir}/${jdk_name}"
 
-  $wso2_service_name  = "wso2${profile}"
   $version            = '2.0.0-ALPHA'
-  $pack               = "wso2-${profile}-${version}"
 
-  # Pack directories
```
Comment (VimukthiPerera) [inline modules/ob_common/manifests/params.pp:37]: If you think this comment is not appropriate, please add a proper comment for the parameters defined here.

File: modules/obbi_dashboard/templates/carbon-home/bin/dashboard.sh.erb [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+###########################################################################
+NAME=start-dashboard
+# Daemon name, where is the actual executable
+
+DASHBOARD_INIT_SCRIPT="$CARBON_HOME/wso2/dashboard/bin/carbon.sh"
+
+# If the daemon is not there, then exit.
+
+$DASHBOARD_INIT_SCRIPT $*
+exit;
```
Comment (VimukthiPerera) [inline modules/obbi_dashboard/templates/carbon-home/bin/dashboard.sh.erb:71]: Please add an empty line to the EoF.

File: modules/obbi_worker/templates/carbon-home/bin/worker.sh.erb [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+###########################################################################
+NAME=start-worker
+# Daemon name, where is the actual executable
+
+WORKER_INIT_SCRIPT="$CARBON_HOME/wso2/worker/bin/carbon.sh"
+
+# If the daemon is not there, then exit.
+$WORKER_INIT_SCRIPT $*
+exit;
```
Comment (VimukthiPerera) [inline modules/obbi_worker/templates/carbon-home/bin/worker.sh.erb:70]: Please add an empty line to the EoF.

File: modules/obbi_worker/templates/carbon-home/deployment/siddhi-files/TRAPaymentValidationApp.siddhi.erb [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+                        NO_OF_TRANSACTIONS string,LOCAL_INSTRUMENT string,AMOUNT string,PAYMENT_SUBMISSION_ID string,
+                        TIMESTAMP long,TRA_DECISION bool, EXEMPTION string, IS_FRAUD bool, MULTI_AUTH_USERS_COUNT int,API_SPEC_VERSION string);
+
+@info(name='getTransactionData')
+from TRAValidationStream as v join PAYMENTS_RAW_DATA as t
+on t.CONSENT_ID == v.CONSENT_ID
+select v.USER_ID, v.MESSAGE_ID, v.CONSENT_ID, t.CLIENT_ID, t.AMOUNT, v.TIMESTAMP as CURRENT_TIMESTAMP, v.TYPE,
+                        v.RECURRING_PAYMENT_TIME, v.MAX_FREQ_WITHOUT_SCA, v.TOTAL_AMOUNT_WITHOUT_SCA,
+                        v.MAX_TRANSACTION_AMOUNT
+insert into TRAPaymentDataStream;
```
Comment (VimukthiPerera) [inline modules/obbi_worker/templates/carbon-home/deployment/siddhi-files/TRAPaymentValidationApp.siddhi.erb:47]: Please make sure that there is an empty line at the end of all files.

File: modules/obbi_worker/templates/carbon-home/deployment/siddhi-files/TRAAccountValidationApp.siddhi.erb [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+from BerlinDataWithElementCountStream [ACCOUNT_LIST_SIZE > 0 or TRANSACTION_LIST_SIZE > 0]
+select TPP_ID, MESSAGE_ID, CONSENT_ID, TIMESTAMP, false as TRA_APPLICABLE, TYPE, LAST_SCA_TIME_LIMIT
+insert into BerlinAccountValidationStream;
+
+@info(name='getBerlinAccountData')
+from BerlinAccountValidationStream as v join ACCOUNTS_RAW_DATA as a
+on a.CONSENT_ID == v.CONSENT_ID
+select v.TPP_ID, v.MESSAGE_ID, v.CONSENT_ID, a.CLIENT_ID, v.TIMESTAMP as CURRENT_TIMESTAMP,  v.TYPE, v.TRA_APPLICABLE,
+                        v.LAST_SCA_TIME_LIMIT
+insert into TRAAccountDataStream;
```
Comment (VimukthiPerera) [inline modules/obbi_worker/templates/carbon-home/deployment/siddhi-files/TRAAccountValidationApp.siddhi.erb:172]: Empty line at EoF.

File: modules/obbi_worker/templates/carbon-home/deployment/siddhi-files/FraudStatusUpdaterApp.siddhi.erb [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+on FraudulentTransactionData.consentId == consentId;
+
+@info(name='updateFraudulentTransactionDataTable')
+from FraudStatusUpdaterStream [option == 0]
+select consentId, option as status
+update FraudulentTransactionData
+set FraudulentTransactionData.status = status
+on FraudulentTransactionData.consentId == consentId;
+
+
```
Comment (VimukthiPerera) [inline modules/obbi_worker/templates/carbon-home/deployment/siddhi-files/FraudStatusUpdaterApp.siddhi.erb:60]: Please remove the extra line.

**Review by VimukthiPerera — CHANGES_REQUESTED** [review]

**Comment by VimukthiPerera** [comment]
Please fix the commit message of the commit 268f997. Since it has more than 50 characters in length limit the commit message to a shorter message and add a description explaining the commit. https://chris.beams.io/posts/git-commit/#limit-50

File: modules/ob_common/manifests/params.pp (hunk 1 above)
Comment (Dinushi) [reply modules/ob_common/manifests/params.pp:37]: I will add a more meaningful comment

File: modules/obbi_dashboard/templates/carbon-home/bin/dashboard.sh.erb (hunk 2 above)
Comment (Dinushi) [reply modules/obbi_dashboard/templates/carbon-home/bin/dashboard.sh.erb:71]: done

File: modules/obbi_worker/templates/carbon-home/bin/worker.sh.erb (hunk 3 above)
Comment (Dinushi) [reply modules/obbi_worker/templates/carbon-home/bin/worker.sh.erb:70]: done

File: modules/obbi_worker/templates/carbon-home/deployment/siddhi-files/TRAPaymentValidationApp.siddhi.erb (hunk 4 above)
Comment (Dinushi) [reply modules/obbi_worker/templates/carbon-home/deployment/siddhi-files/TRAPaymentValidationApp.siddhi.erb:47]: done

File: modules/obbi_worker/templates/carbon-home/deployment/siddhi-files/TRAAccountValidationApp.siddhi.erb (hunk 5 above)
Comment (Dinushi) [reply modules/obbi_worker/templates/carbon-home/deployment/siddhi-files/TRAAccountValidationApp.siddhi.erb:172]: done

File: modules/obbi_worker/templates/carbon-home/deployment/siddhi-files/FraudStatusUpdaterApp.siddhi.erb (hunk 6 above)
Comment (Dinushi) [reply modules/obbi_worker/templates/carbon-home/deployment/siddhi-files/FraudStatusUpdaterApp.siddhi.erb:60]: done

File: modules/ob_common/manifests/params.pp (hunk 1 above)
Comment (Dinushi) [reply modules/ob_common/manifests/params.pp:37]: done

File: modules/obbi_worker/templates/carbon-home/deployment/siddhi-files/TRAAccountValidationApp.siddhi.erb (hunk 5 above)
Comment (Dinushi) [reply modules/obbi_worker/templates/carbon-home/deployment/siddhi-files/TRAAccountValidationApp.siddhi.erb]: done

**Comment by Dinushi** [comment]
> Please fix the commit message of the commit [268f997](https://github.com/wso2/puppet-open-banking/commit/268f9972cb121a1db2a6332c9445f9c0eb2af163). Since it has more than 50 characters in length, limit the commit message to a shorter message and add a description explaining the commit. https://chris.beams.io/posts/git-commit/#limit-50

**Review by VimukthiPerera — APPROVED** [review]

**Review by DivyaPremanantha — APPROVED** [review]

File: modules/obbi_dashboard/templates/carbon-home/bin/dashboard.sh.erb [hunk 7]
```diff
@@ -0,0 +1,71 @@
+#!/bin/sh
+# ---------------------------------------------------------------------------
+#  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (Ashi1993) [inline modules/obbi_dashboard/templates/carbon-home/bin/dashboard.sh.erb:3]: Change the year to 2020

File: modules/obbi_dashboard/templates/carbon-home/conf/dashboard/deployment.yaml.erb [hunk 8]
```diff
@@ -0,0 +1,462 @@
+################################################################################
+#   Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved
```
Comment (Ashi1993) [inline modules/obbi_dashboard/templates/carbon-home/conf/dashboard/deployment.yaml.erb:2]: Change the year

File: modules/obbi_dashboard/templates/carbon-home/deployment/siddhi-files/FraudStatusUpdaterApp.siddhi.erb [hunk 9]
```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.com). All Rights Reserved.
```
Comment (Ashi1993) [inline modules/obbi_dashboard/templates/carbon-home/deployment/siddhi-files/FraudStatusUpdaterApp.siddhi.erb:2]: change the year

File: modules/obbi_dashboard/templates/carbon-home/deployment/siddhi-files/TRAAccountValidationApp.siddhi.erb [hunk 10]
```diff
@@ -0,0 +1,172 @@
+/**
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.com). All Rights Reserved.
```
Comment (Ashi1993) [inline modules/obbi_dashboard/templates/carbon-home/deployment/siddhi-files/TRAAccountValidationApp.siddhi.erb:2]: change the year on all places

**Pull request merged**